sdk/resourcemanager/azure-resourcemanager-containerinstance/src/main/java/com/azure/resourcemanager/containerinstance/models/ContainerGroup.java

```diff
@@ -84,6 +84,9 @@ public interface ContainerGroup extends GroupableResource<ContainerInstanceManag
     /** @return the DNS configuration for the container group */
     DnsConfiguration dnsConfig();
 
+    /** @return the reuse policy of DSN name label */
```

Comment:
javadoc need to write full description https://github.com/Azure/azure-sdk-for-java/issues/36596

Reply:
I mean, all javadoc has to have a "description" part.

```java
/**
 * <description>
 *
 * @return: ...
...
```

Reply:
Fixed in the [commit#a9bf1bf](https://github.com/Azure/azure-sdk-for-java/pull/42795/commits/a9bf1bf61dfd459b0e3e909ae283257cb6b337b8)